testsuite/features/secondary/srv_mainpage.feature

```diff
@@ -19,23 +19,6 @@ Feature: Main landing page options and preferences
     And I follow "API Documentation"
     Then I should see a "API Overview" text
 
-# TODO: We need to refactor this to navigate to a doc page
```

Comment:
I'm reading now this TODO, and I wonder if we have scenarios for this somewhere else.
🤔  @bisht-richa do you remember about this?

Reply:
Yes, we need this for Copyright Notice